src/main/java/com/afornalik/ox/board/BoardField.java

```diff
@@ -1,14 +1,30 @@
 package com.afornalik.ox.board;
 
-class BoardField {
+import java.util.Objects;
+
+public class BoardField {
 
     private final FieldStatus fieldStatus;
 
     BoardField(FieldStatus fieldStatus) {
         this.fieldStatus = fieldStatus;
     }
 
-    FieldStatus showStatus() {
+    public FieldStatus showStatus() {
         return fieldStatus;
     }
```

Comment:
So, basically you're wrapping one enum field with an object? Is it really necessary? I'm serious, for me it's an overkill.

Reply:
Also, does this class need to be public? You "use it" outside of the package only once in console ui, and that should be easy to walkaround.

Reply:
Yes true, i had some idea when i created it but, finally it's a pointless class - deleted.